build/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM registry.access.redhat.com/ubi8/ubi
-RUN curl https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo > /etc/yum.repos.d/bazel.repo && \
-    yum install -y bazel && \
-    yum clean all
+
+RUN dnf install -y dnf-plugins-core && \
+       dnf copr -y enable vbatts/bazel && \
+       dnf install -y bazel
```

Comment:
What's the effect on image size with doing it in this way? Also, should we `dnf clean all` as well?

Reply:
It might not even be necessary to install bazel. See my comment here: https://github.com/che-incubator/che-workspace-crd-operator/pull/3/files#r363647973